tests/e2e/docs/supervisor_cluster_setup.md

```diff
@@ -98,6 +98,10 @@ datacenters should be comma separated if deployed on multi-datacenters
     export SHARED_NFS_DATASTORE_URL="<shared-NFS-datastore-url>"
     #shared VMFS datastore url
     export SHARED_VMFS_DATASTORE_URL="<shared-VMFS-datastore-url>"
+    #For vsan direct tests for spbm policy driven allocation tests, set following variables
+    export DATASTORE_TYPE="VSAN_DIRECT"
```

Comment:
Instead we can use `USE_VSAN_DIRECT_DATASTORE_IN_WCP`. `DATASTORE_TYPE` applies to other cases as well, it will be difficult to control its usage.

Reply:
Done